lib/client/api.go

```diff
@@ -1121,7 +1121,7 @@ func (c *Config) ResourceFilter(kind string) *proto.ListResourcesRequest {
 }
 
 // DTAuthnRunCeremonyFunc matches the signature of [dtauthn.Ceremony.Run].
-type DTAuthnRunCeremonyFunc func(context.Context, devicepb.DeviceTrustServiceClient, *devicepb.UserCertificates) (*devicepb.UserCertificates, error)
+type DTAuthnRunCeremonyFunc func(ctx context.Context, clt devicepb.DeviceTrustServiceClient, sshCert []byte, sshSigner *keys.PrivateKey) (*devicepb.UserCertificates, error)
```

Comment:
A few remarks here:

1. nit: Avoid `ctl`. Maybe `devicesClient` instead, if we do keep the named params.
2. I would still keep `*devicepb.UserCertificates` as the input, as it is the stronger type (even though not fully used).
3. Could we pass a `crypto.Signer` instead of a `*keys.PrivateKey`? Or is the PrivateKey type necessary?
4. Finally, we could consider an "input struct" as the number of params grow, for clarity / future-proofing.

Reply:
switched to `crypto.Signer`, reverted to `*devicepb.UserCertificates`, and pulled these out into an input struct. I don't really like the name `CeremonyRunParams` but I was struggling to think of anything better, very easy for me to rename if you have a better suggestion